parent-pom/pom.xml

```diff
@@ -40,6 +40,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <gpg.skip>true</gpg.skip>
+        <test.excludeTags/>
```

Comment:
Explicitly specify property in parent pom. Won't be defined in generated parent poms any more.